services/core/ForwardHistorian/config

```diff
@@ -2,6 +2,7 @@
 
     "agentid": "forwarder",
     "destination-vip": "ipc://@/home/volttron/.volttron/run/vip.socket",
+    "destination-serverkey": "",
```

Comment:
This should be null rather than "" in configuration file. Otherwise it will be passed as an empty string which is an invalid server key.